test/WhoopsErrorHandlerTest.php

```diff
@@ -15,6 +15,7 @@
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 use Psr\Http\Message\StreamInterface;
+use ReflectionClass;
```

Comment:
As this file is changed, year in header license notes should be updated to 2017.